libcloudforensics/providers/aws/internal/account.py

```diff
@@ -428,6 +429,9 @@ def CreateVolumeFromSnapshot(
       snapshot (AWSSnapshot): Snapshot to use.
       volume_name (str): Optional. String to use as new volume name.
       volume_name_prefix (str): Optional. String to prefix the volume name with.
+      volume_type (str): Optional. The volume type for the volume to create.
+          Can be one of 'standard'|'io1'|'gp2'|'sc1'|'st1'. The default is
+          'gp2'.
```

Comment:
Should we raise an exception if the volume type is not one of these?

Reply:
Good idea, done